### PR TITLE
feat(secgroup): support the new parameters

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
Support the new parameters for ver.3 API, the parameters are as follows:
+ remote_address_group_id
+ action
+ priority

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. support three new parameters
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccNetworkingSecGroupRule_remoteAddressGroup'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run=TestAccNetworkingSecGroupRule_remoteAddressGroup -timeout 360m -parallel 4
=== RUN   TestAccNetworkingSecGroupRule_remoteAddressGroup
=== PAUSE TestAccNetworkingSecGroupRule_remoteAddressGroup
=== CONT  TestAccNetworkingSecGroupRule_remoteAddressGroup
--- PASS: TestAccNetworkingSecGroupRule_remoteAddressGroup (45.49s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       45.640s
```
```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccNetworkingSecGroupRule_action'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run=TestAccNetworkingSecGroupRule_action -timeout 360m -parallel 4
=== RUN   TestAccNetworkingSecGroupRule_action
=== PAUSE TestAccNetworkingSecGroupRule_action
=== CONT  TestAccNetworkingSecGroupRule_action
--- PASS: TestAccNetworkingSecGroupRule_action (38.19s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       38.274s
```
```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccNetworkingSecGroupRule_priority'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run=TestAccNetworkingSecGroupRule_priority -timeout 360m -parallel 4
=== RUN   TestAccNetworkingSecGroupRule_priority
=== PAUSE TestAccNetworkingSecGroupRule_priority
=== CONT  TestAccNetworkingSecGroupRule_priority
--- PASS: TestAccNetworkingSecGroupRule_priority (37.80s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       37.879s
```

### CODE REVIEW

File: huaweicloud/resource_huaweicloud_networking_secgroup_rule.go [hunk 1]
```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
 )
 
+var unsupportedParams = []string{"ports", "remote_address_group_id", "action", "priority"}
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_networking_secgroup_rule.go]: unsupported or advanced?

File: docs/resources/networking_secgroup_rule.md [hunk 2]
```diff
@@ -64,9 +65,18 @@ The following arguments are supported:
...
+* `remote_group_id` - (Optional, String, ForceNew) Specifies the remote group ID. Changing this creates a new security
   group rule.
 
+* `remote_address_group_id` - (Optional, String, ForceNew) Specifies the remote address group ID.
+  Changing this creates a new security group rule.
+
+* `action` - (Optional, String, ForceNew) Specifies the effective policy. The valid values are **allow** and **deny**.
+  Changing this creates a new security group rule.
+
+* `priority` - (Optional, String, ForceNew) Specifies the priority number.
```
Comment (ShiChangkuo) [inline docs/resources/networking_secgroup_rule.md:77]: please add valide value range

File: huaweicloud/resource_huaweicloud_networking_secgroup_rule.go (hunk 1 above)
Comment (Lance52259) [reply huaweicloud/resource_huaweicloud_networking_secgroup_rule.go]: advanced is good

File: docs/resources/networking_secgroup_rule.md (hunk 2 above)
Comment (Lance52259) [reply docs/resources/networking_secgroup_rule.md:77]: done.

**Comment by Lance52259** [comment]
```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccNetworkingSecGroupRule_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run=TestAccNetworkingSecGroupRule_ -timeout 360m -parallel 4
=== RUN   TestAccNetworkingSecGroupRule_basic
=== PAUSE TestAccNetworkingSecGroupRule_basic
=== RUN   TestAccNetworkingSecGroupRule_oldPorts
=== PAUSE TestAccNetworkingSecGroupRule_oldPorts
=== RUN   TestAccNetworkingSecGroupRule_remoteGroup
=== PAUSE TestAccNetworkingSecGroupRule_remoteGroup
=== RUN   TestAccNetworkingSecGroupRule_lowerCaseCIDR
=== PAUSE TestAccNetworkingSecGroupRule_lowerCaseCIDR
=== RUN   TestAccNetworkingSecGroupRule_noPorts
=== PAUSE TestAccNetworkingSecGroupRule_noPorts
=== RUN   TestAccNetworkingSecGroupRule_remoteAddressGroup
=== PAUSE TestAccNetworkingSecGroupRule_remoteAddressGroup
=== RUN   TestAccNetworkingSecGroupRule_action
=== PAUSE TestAccNetworkingSecGroupRule_action
=== RUN   TestAccNetworkingSecGroupRule_priority
=== PAUSE TestAccNetworkingSecGroupRule_priority
=== CONT  TestAccNetworkingSecGroupRule_basic
=== CONT  TestAccNetworkingSecGroupRule_noPorts
=== CONT  TestAccNetworkingSecGroupRule_remoteGroup
=== CONT  TestAccNetworkingSecGroupRule_lowerCaseCIDR
--- PASS: TestAccNetworkingSecGroupRule_basic (54.47s)
=== CONT  TestAccNetworkingSecGroupRule_oldPorts
--- PASS: TestAccNetworkingSecGroupRule_remoteGroup (55.41s)
=== CONT  TestAccNetworkingSecGroupRule_action
--- PASS: TestAccNetworkingSecGroupRule_noPorts (56.94s)
=== CONT  TestAccNetworkingSecGroupRule_priority
--- PASS: TestAccNetworkingSecGroupRule_lowerCaseCIDR (56.97s)
=== CONT  TestAccNetworkingSecGroupRule_remoteAddressGroup
--- PASS: TestAccNetworkingSecGroupRule_oldPorts (49.17s)
--- PASS: TestAccNetworkingSecGroupRule_priority (50.19s)
--- PASS: TestAccNetworkingSecGroupRule_remoteAddressGroup (52.22s)
--- PASS: TestAccNetworkingSecGroupRule_action (55.49s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       111.105s
```

**Pull request merged**